lib/features/overview/widgets/givt_list_item.dart

```diff
@@ -40,6 +40,9 @@ class GivtListItem extends StatelessWidget {
     final showCancelButton = givtGroup.status == 1;
     final showRefundButton = givtGroup.status == 3 && isWithinLastMonth;
 
+    // Check if this is an online giving donation (type 7)
+    final isOnlineGiving = givtGroup.status == 7;
```

Comment:
You didn't test this I guess? =)
Status is something else then (transaction)-type

Reply:
Tested it using == true, i misunderstood then what online giving was, will fix